src/components/pages/dashboard-view/DashboardViewContainer.jsx

```diff
@@ -33,13 +33,15 @@ const DashboardViewContainer = () => {
       key="app-container"
     >
       {programs.map(program => (
-        <GhostLink to={`/program/${program[0].programId}`}>
+        <GhostLink to={`/program/view/${program[0].programId}`}>
```

Comment:
Nicely done, looks great. Awesome to see the split path for viewing and editing added here!